opengapps-packages.mk

```diff
@@ -1,3 +1,4 @@
+include vendor/google/build/config.mk
```

Comment:
Hi, I agree it looks like it should work without that line; I just double checked by removing it and nothing from opengapps-files gets included in the build and adding the following just prints an empty string
```
$(warning $(GAPPS_FILES))
```